content/menu/index.md

```diff
@@ -0,0 +1,47 @@
+---
```

Comment:
What does the menu look like if we let the theme auto-populate it from the repository directory structure?